gcloud/storage/connection.py

```diff
@@ -256,7 +256,7 @@ def api_request(self, method, path, query_params=None,
             content_type = response.get('content-type', '')
             if not content_type.startswith('application/json'):
                 raise TypeError('Expected JSON, got %s' % content_type)
-            return json.loads(content)
+            return json.loads(content.decode('utf-8'))
```

Comment:
@tseaver `content.decode('utf-8')` works when content is a Python 2 `str` and `unicode` and Python 3 `bytes`.